tests/catalog.tests.bom

```diff
@@ -0,0 +1,97 @@
+brooklyn.catalog:
+  version: 0.8-SNAPSHOT
+  iconUrl: https://raw.githubusercontent.com/cloudsoft/brooklyn-hyperledger/master/hyperledger-fabric-icon.png
+  license_code: Apache-2.0
+  license_url: http://www.apache.org/licenses/LICENSE-2.0.txt
+  dependsOn:
+  # From https://github.com/brooklyncentral/common-catalog-utils
+  - commontests/common.tests.bom
+
+  items:
+  - "https://raw.githubusercontent.com/brooklyncentral/common-catalog-utils/master/common-tests/src/main/resources/commontests/common.tests.bom"
+
+  - id: hyperledger-fabric-single-cluster-application-test
+    itemType: template
+    name: Hyperledger Cluster Test
+    item:
+      services:
+      - type: hyperledger-fabric-single-cluster-application
+        id: target-app
+
+      - type: test-case
+        name: Hyperledger assertions
+        brooklyn.config:
+          targetId: target-app
+          timeout: 30m
+          timeout.initialStartup: 1h
+        brooklyn.children:
+
+        - type: assert-up-and-running-initial
+
+        - type: invoke-effector
+          name: Example blockchain app
+          brooklyn.config:
+            targetId: my-hyperledger-cli-node
+            effector: Run Demo Application
+            # TODO Want to assert output contains "Dave is the owner!", but that is in the shh stderr
```

Comment:
@mikezaccardo I wondered about doing that. I preferred to test the effector, given we're defining an effector (otherwise the effector never gets tested).

It would be nice to have the effector response show the right output. Not sure why "Dave is the owner" now goes to stderr, whereas your and Sam's previous comments in the code suggested it went to stdout for you.